lib/srv/db/redis/cmds.go

```diff
@@ -0,0 +1,211 @@
+/*
+
+ Copyright 2022 Gravitational, Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+*/
+
+package redis
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/gravitational/teleport/lib/defaults"
+	"github.com/gravitational/teleport/lib/services"
+	"github.com/gravitational/teleport/lib/srv/db/common"
+	"github.com/gravitational/teleport/lib/srv/db/common/role"
+	"github.com/gravitational/trace"
+)
+
+// List of all commands that Teleport handles in a special way.
+const (
+	helloCmd        = "hello"
+	authCmd         = "auth"
+	subscribeCmd    = "subscribe"
+	psubscribeCmd   = "psubscribe"
+	punsubscribeCmd = "punsubscribe"
+	ssubscribeCmd   = "ssubscribe"
+	sunsubscribeCmd = "sunsubscribe"
+)
+
+const defaultUsername = "default"
```

Comment:
Move to a shared constants in lib/defaults (it's already used elsewhere as well, IIRC in CLI).

Reply:
Moved